pycode/memilio-epidata/memilio/epidata/getPopulationData.py

```diff
@@ -535,8 +532,8 @@ def get_population_data(read_data=dd.defaultDict['read_data'],
             filename = 'county_current_population'
             filename_raw = 'county_population'
         else:  # Write Dataframe without merging
-            filename = 'county_current_population_dim401'
-            filename_raw = 'county_population_dim401'
+            filename = 'county_current_population_dim' + str(len(df_current))
+            filename_raw = 'county_population_dim' + str(len(df_current))
```

Comment:
Maybe print a warning if dimension is not 400 or 401 (which should now not be possible)